ion-schema/src/isl/mod.rs

```diff
@@ -577,6 +577,16 @@ mod isl_tests {
         Range::from_ion_element(
             &Element::read_one(text.as_bytes()).expect("parsing failed unexpectedly"),
             RangeType::Any,
+            IslVersion::V1_0,
+        )
+    }
+
+    // helper function to create a range
+    fn load_range_v2_0(text: &str) -> IonSchemaResult<Range> {
+        Range::from_ion_element(
+            &Element::read_one(text.as_bytes()).expect("parsing failed unexpectedly"),
+            RangeType::Any,
+            IslVersion::V2_0,
```

Comment:
The `load_range_...` functions appear to be private functions, so you could combine these functions and just make the ISL version an argument to the function.

Reply:
Makes sense. Will change that.